run/types/testing.ts

```diff
@@ -348,4 +358,5 @@ export type Id =
   | 'Image picker'
   | 'network.loki.messenger:id/action_apply'
   | 'Save'
-  | 'Error message';
```

Comment:
Something fishy happened here in the first place for Error message to end up in the IDs instead of the Accessibility IDs. The locators/onboarding.ts (line 17) expects it to be accessibility id. 
So yes it's an intentional change.